tests/serial/rbf-parameter-tuning/GaussianShapeParameter.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <data:vector name="DataOne" />
+
+  <mesh name="MeshOne" dimensions="3">
+    <use-data name="DataOne" />
+  </mesh>
+
+  <mesh name="MeshTwo" dimensions="3">
+    <use-data name="DataOne" />
+  </mesh>
+
+  <participant name="SolverOne">
+    <provide-mesh name="MeshOne" />
+    <write-data name="DataOne" mesh="MeshOne" />
+  </participant>
+
+  <participant name="SolverTwo">
+    <receive-mesh name="MeshOne" from="SolverOne" />
+    <provide-mesh name="MeshTwo" />
+    <mapping:rbf-global-direct
```

Comment:
The same solver backend is also involved in the `pu-rbf` solver, but the test doesnt cover this yet.